pages/graphql.vue

```diff
@@ -224,6 +319,7 @@ export default {
           method: "post",
           url: this.url,
           headers: {
+            ...this.headers,
```

Comment:
I couldn't spot headers on network tab, instead its index are appended 😄 

to reproduce:
0. Added few headers and its values
1. Get Scheme
2. Check request headers and see only its index are appended. Ex: 0, 1, 2...

![Screen Shot 2019-11-26 at 9 39 47 AM](https://user-images.githubusercontent.com/10395817/69599041-935cdd00-1031-11ea-872b-159e1e3c5762.png)

Reply:
@liyasthomas I fixed it in a later commit...

Check it out...